src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_command_type.py

```diff
@@ -31,5 +31,5 @@ def cli_storage_data_plane_command(name, operation, client_factory,
     command.add_argument('sas_token', '--sas-token', required=False, default=None,
                          arg_group=group_name,
                          help='A Shared Access Signature (SAS). Must be used in conjunction with '
-                         'storage account name. Var: AZURE_SAS_TOKEN')
+                         'storage account name. Var: AZURE_STORAGE_SAS_TOKEN')
```

Comment:
>Var: [](start = 48, length = 4)

we might want to spell this out since var isn't necessarily obvious at first read